test/passing/structure_item_grouping.ml.opts

```diff
@@ -0,0 +1 @@
+--max-iter=3
```

Comment:
This is needed since the implementation based on checking locations requires reformatting the whole file again when something is changed to or from a one-liner.

Reply:
Thanks! Did not think of this.